### ISSUE TITLE
Shape files imported do not show up under recent files in the data viewer

### ISSUE THREAD

**Issue by Patowhiz** [africanmathsinitiative/R-Instat#7511]
When I import a shape file, then close the data book. It does not show in the list of recent files. Even when I reopen R-Instat.
I would expect shape files just like any other files to be shown in the list of recent files.

**Comment by rdstern** [comment]
I agree.  But I think if it does show, then it will also need the "ordinary" File > Import to be used for the importing.  Which will be nice, but perhaps not as urgent as other tasks just now?  This is the same with netCDF files I think, which also are imported through a different dialogue.

**Comment by Patowhiz** [comment]
@rdstern I agree

Just to clarify, this only happens when you use;
![shapefile](https://user-images.githubusercontent.com/21179007/170986417-a476d9de-b571-4750-a773-921ad6a35338.png)
